ffi/freetype_h.lua

```diff
@@ -19,18 +20,18 @@ struct FT_ListRec_ {
   FT_ListNode tail;
 };
 struct FT_BBox_ {
-  long int xMin;
-  long int yMin;
-  long int xMax;
-  long int yMax;
+  FT_Pos xMin;
```

Comment:
This is because of the ffi-cdecl last month?

Reply:
No, I think it's because I added FT_Pos. Before, it could just masquerade as long int without annoucing its real name :)